deltacat/catalog/iceberg/impl.py

```diff
@@ -9,6 +9,10 @@
 from deltacat.exceptions import TableAlreadyExistsError
 from deltacat.storage.iceberg.model import PartitionSchemeMapper, SchemaMapper
 from deltacat.storage.model.partition import PartitionScheme
+from deltacat.storage.model.push_down import Pushdown
+from deltacat.storage.model.scan_plan import ScanPlan
+from deltacat.storage.model.scan_task import FileScanTask, DataFile
+from deltacat.storage.iceberg.impl import _try_load_iceberg_table
```

Comment:
Instead of bringing in this protected method from `storage`, can we move `create_scan_plan` down to the `storage` layer?